src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -1079,6 +1081,16 @@ public static String prefixName(String name) {
     return PDAO_PREFIX + name;
   }
 
+  private Schema buildRowMetadataSchema() {
+    List<Field> fieldList = new ArrayList<>();
+    fieldList.add(Field.of(PDAO_ROW_ID_COLUMN, LegacySQLTypeName.STRING));
+    fieldList.add(
+        Field.newBuilder("load_time", LegacySQLTypeName.TIMESTAMP)
+            .setMode(Field.Mode.REQUIRED)
+            .build());
+    return Schema.of(fieldList);
```

Comment:
Not a huge deal but since the list is fixed you don't need to `new ArrayList()` and call `List.add()`, you can use a fixed list.
```suggestion
    List<Field> fieldList = List.of(
        Field.of(PDAO_ROW_ID_COLUMN, LegacySQLTypeName.STRING),
        Field.newBuilder("load_time", LegacySQLTypeName.TIMESTAMP)
            .setMode(Field.Mode.REQUIRED)
            .build());
    return Schema.of(fieldList);
```